tests/integration/tests.cake

```diff
@@ -56,8 +56,41 @@ Task("CheckBinaryFilesTrackedByLfs")
         reportFilePath);
 });
 
+Task("CheckFilesPathLength")
+    .Does(() =>
+{
+    var repoRootDir = MakeAbsolute(Directory("../../"));
+
+    var issues =
+        ReadIssues(
+            GitRepositoryIssuesAliases.GitRepositoryIssues(
+                Context,
+                new GitRepositoryIssuesSettings
+                {
+                    CheckFilesPathLength = true,
+                    FilesPathLength = 60
+                }),
+            new ReadIssuesSettings(repoRootDir)
+            {
+                 Format = IssueCommentFormat.Html
+            });
+
+    var reportDir =
+        repoRootDir.Combine("BuildArtifacts").Combine("TestResults").Combine("Integration");
+    var reportFilePath =
+        reportDir.CombineWithFilePath("issues-report.html");
```

Comment:
Should use a different file name than the `CheckBinaryFilesTrackedByLfs` test case, otherwise the first report will be overriden.

Reply:
Ok, I'll change that.